src/demo/containers/App.js

```diff
@@ -207,6 +210,86 @@ class App extends Component {
           <SynapseTable>
           </SynapseTable>
         </QueryWrapper>
+
+        <QueryWrapper
+          initQueryRequest={{
```

Comment:
add ability to give you the Query Bundle Response (for Synapse Tables/governance issue)